AI-and-Analytics/Getting-Started-Samples/INC-Quantization-Sample-for-PyTorch/README.MD

```diff
@@ -51,58 +40,86 @@ The sample contains one Jupyter Notebook and one Python script.
- 
-#### Activate Conda
-
-1. Activate the Conda environment.
-
-    ```
-    conda activate pytorch
-    ```
-
-   By default, the Intel AI Tools are installed in the `/opt/intel/oneapi` folder and require root privileges to manage them.
-
-   You can choose to activate Conda environment without root access. To bypass root access to manage your Conda environment, clone and activate your desired Conda environment using the following commands similar to the following.
+You will need to download and install the following toolkits, tools, and components to use the sample.
 
-### Run the Notebook
+**1. Get Intel® AI Tools**
 
-1. Launch Jupyter Notebook.
-   ```
-   jupyter notebook --ip=0.0.0.0
-   ```
-2. Follow the instructions to open the URL with the token in your browser.
-3. Locate and select the Notebook.
-   ```
-   optimize_pytorch_models_with_ipex.ipynb
-   ```
-4. Change the kernel to **pytorch**.
-5. Run every cell in the Notebook in sequence.
+Required AI Tools: **Intel® Neural Compressor, Intel® Extension of PyTorch***. 
+<br>Select and install needed AI Tools via [AI Tools Selector](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html). AI and Analytics samples are validated on AI Tools Offline Installer. It is recommended to select Offline Installer option in AI Tools Selector.
 
-#### Troubleshooting
+**2. Install dependencies**
+```
+pip install -r requirements.txt
+```
+**Install Jupyter Notebook** by running `pip install notebook`. Alternatively, see [Installing Jupyter](https://jupyter.org/install) for detailed installation instructions.
 
-If you receive an error message, troubleshoot the problem using the **Diagnostics Utility for Intel® oneAPI Toolkits**. The diagnostic utility provides configuration and system checks to help find missing dependencies, permissions errors, and other issues. See the [Diagnostics Utility for Intel® oneAPI Toolkits User Guide](https://www.intel.com/content/www/us/en/develop/documentation/diagnostic-utility-user-guide/top.html) for more information on using the utility.   
+## Run the `Getting Started with Intel® Neural Compressor for Quantization` Sample 
 
+>**Note**: Before running the sample, make sure [Environment Setup](https://github.com/oneapi-src/oneAPI-samples/tree/master/AI-and-Analytics/Getting-Started-Samples/INC-Quantization-Sample-for-PyTorch#environment-setup) is completed.
+
+Go to the section which corresponds to the installation method chosen in [AI Tools Selector](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html) to see relevant instructions:
+* [AI Tools Offline Installer (Validated)](https://github.com/oneapi-src/oneAPI-samples/tree/master/AI-and-Analytics/Getting-Started-Samples/INC-Quantization-Sample-for-PyTorch#ai-tools-offline-installer-validated)
```

Comment:
@Marina-Zubova Can we change the link to `#ai-tools-offline-installer-validated`? I think that way the link will work and we can test it out before we merge to the master branch.